<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLineArcArcArcLine</class>
 <widget class="QDialog" name="CreateLineArcArcArcLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create-Line-Arc-Arc-Arc-Line</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The order of created alignment points has to be according to drive-direction.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>                                                         R2=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>your third created point will be the endpoint of arc3. Unchecked RAL-K-1 can not</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_R2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>If RAL-K-1 is unchecked, your first created point will be the startpoint of arc1 and</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_ral">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RAL-K-1 (a1=17.5 gon, a2=22.5 gon)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_noral">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>no RAL-K-1</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_213">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>104</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R1:R2:R3=2:1:3</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_25155">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>316</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>R1:R2:R3=2.5:1:5.5</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>generate a solution for every angle and linelength. Therefore, if posible, use RAL-K-1.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
